src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
-export interface Figure {
+type FigureType = 'circle' | 'rectangle' | 'triangle';
+type FigureColor = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: FigureType;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
fix it in all places
```suggestion
    public color: FigureColor,
```